docs/creating-service-bindings/service-binding-resource.md

```diff
@@ -4,8 +4,122 @@ sidebar_position: 6
 
 # Service Binding Resource
 
-_In this section, we would highlight the different options based on the specification_
+## Service Binding Resource
 
-https://github.com/k8s-service-bindings/spec#service-binding
+Service Binding resources use `binding.operators.coreos.com` group with  `v1alpha1` version and `ServiceBinding` kind.
 
-_ We would refer to the specifcation about the Resource Type Schema_
\ No newline at end of file
+A sample Service Binding resource that connects a nodejs application to a postgresql database:
+
+
+```yaml
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+
+metadata:
+  name: binding-request 
+
+spec:
+
+  application:
+    name: nodejs-rest-http-crud
+    group: apps
+    version: v1
+    resource: deployments
+
+  services:
+  - group: postgresql.dev
+    version: v1alpha1
+    kind: Database
+    name: db-demo
+```
+
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| .apiVersion | string | APIVersion defines the versioned schema of this representation of an object |
+| .kind | string | Kind is a string value representing the REST resource this object represents |
+| .metadata.name | string | Name is primarily intended for creation idempotence and configuration definition. It cannot be updated. It must be unique within a namespace |
+| .metadata.namespace | string | Namespace of the resource |
+| .spec | object | Spec describes the application and services |
+| .status | object | Current status of the service binding |
+
+## Service Binding Spec
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| .spec.application | array (string) | It is used to uniquely identify an application resource |
+| .spec.application.group | string | It is the API group of the application resource |
+| .spec.application.version | string | It is the API group version of the application resource |
+| .spec.application.resource | string | It is the type of the application resource. It can be collected from `.kind` field of the application resource |
```

Comment:
We actually refer a k8s resource here, using GroupVersionResource concept well established in k8s - perhaps linking to https://book.kubebuilder.io/cronjob-tutorial/gvks.html or something similar from the official doc would be easier? For example: https://book.kubebuilder.io/cronjob-tutorial/gvks.html or https://kubernetes.io/docs/reference/using-api/api-concepts/ ?